src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -73,15 +73,12 @@ AutomatedRssDownloader::AutomatedRssDownloader(const QWeakPointer<Rss::Manager>
     Q_ASSERT(ok);
     m_ruleList = manager.toStrongRef()->downloadRules();
     m_editableRuleList = new Rss::DownloadRuleList; // Read rule list from disk
-    m_episodeRegex = new QRegExp("^(^\\d{1,4}x(\\d{1,4}(-(\\d{1,4})?)?;){1,}){1,1}",
-                                 Qt::CaseInsensitive);
     QString tip = "<p>" + tr("Matches articles based on episode filter.") + "</p><p><b>" + tr("Example: ")
```

Comment:
Episode regex used for validating removed. I see you made it a static in the function - I'm OK with that since it's not used elsewhere.